scripts/partition_manager/partition_manager.rst

```diff
@@ -812,6 +812,37 @@ You can add or remove partitions as described in the following sections.
 .. note::
   If the static configuration contains an entry for the ``app`` partition, this entry is ignored.
 
+.. _ug_pm_config_dynamic:
+
+Configuring dynamic partitions
+==============================
+
+For child images that use dynamic partition sizing, the Partition Manager allows you to use the ``CONFIG_PM_PARTITION_SIZE_<CHILD_IMAGE>`` Kconfig option to adjust the partition size.
+This is particularly useful when you want to optimize the memory usage of MCUboot to less than 16 kB by enabling cryptographic functionalities.
```

Comment:
I would not mention particular size (16 kB) as it might be not achievable. Maybe 'the smallest possible size` or so.
The smallest possible size might vary form release to release.